linters/eslint/test_data/eslint_v8.10.0_bad_install.check.shot

```diff
@@ -6,10 +6,12 @@ Object {
   "lintActions": Array [],
   "taskFailures": Array [
     Object {
+      "details": Any<String>,
       "message": "test_data/eof_autofix.ts... (4 files)",
       "name": "eslint",
     },
     Object {
+      "details": Any<String>,
```

Comment:
Why use an any here? Can this be a discriminated union instead?

Reply:
Not necessary for https://jestjs.io/docs/snapshot-testing#property-matchers